### PR TITLE
Add G4HepEm to CMSSW

### PR DESCRIPTION
**Pull request by hahnjo**
@smuzaffar @civanch

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

File: g4hepem.spec [hunk 1]
```diff
@@ -0,0 +1,50 @@
+### RPM external g4hepem 20221014
+## INCLUDE compilation_flags
+%define tag %{realversion}
+%define branch master
+%define github_user mnovak42
+Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/v%{tag}&export=%{n}.%{realversion}&output=/%{n}.%{realversion}-%{tag}.tgz
```
Comment (smuzaffar) [inline g4hepem.spec]: @hahnjo , there is `v20221014` tag in the github repo, so either create `v20221014` tag or update the above line
```suggestion
Source: git+https://github.com/%github_user/%{n}.git?obj=%{branch}/%{tag}&export=%{n}.%{realversion}&output=/%{n}.%{realversion}-%{tag}.tgz
```

Comment (hahnjo) [reply g4hepem.spec]: Gnah, I didn't see this one...

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
test parameters:
- pull_request = cms-sw/cmssw#39736

**Comment by smuzaffar** [comment]
please test for el8_aarch64_gcc10

**Comment by smuzaffar** [comment]
please test for el8_ppc64le_gcc10

**Comment by smuzaffar** [comment]
please test for el8_amd64_gcc11

**Comment by smuzaffar** [comment]
+externals
thanks @hahnjo this looks good

**Pull request merged**